app/models/spree/tracker_decorator.rb

```diff
@@ -1,7 +1,8 @@
 Spree::Tracker.class_eval do
   belongs_to :store
 
-  def self.current(domain)
+  def self.current(domain=:google_analytics)
```

Comment:
@VygovskySergey this needs to be `current(engine = TRACKING_ENGINES.first, domain)`

see https://github.com/spree-contrib/spree_analytics_trackers/blob/master/app/models/spree/tracker.rb#L12